apps/examples/webserver/webserver_main.c

```diff
@@ -99,12 +102,30 @@ static const char g_httpcnlost[] = "close";
 struct http_server_t *http_server = NULL;
 struct http_server_t *https_server = NULL;
 
+static void http_send_chunk(struct http_client_t *client)
+{
+	char msg_body[MAX_DATA_SIZE + 1] = {0, };
+
+	//add data
+	memset(msg_body, 0, MAX_DATA_SIZE + 1 );
+	memset(msg_body, 'a', MAX_DATA_SIZE);
+
+	http_send_response_chunk(client, 200, "OK", msg_body, NULL, FIRST_DATA);
+
+	for(int i = 0; i < MAX_DATA_COUNT; i++) {
```

Comment:
Please use the same policy for `( )` in the same file.
```
for (
```
or
```
for(
```

Reply:
done